src/bin/cargo/commands/test.rs

```diff
@@ -38,12 +38,12 @@ pub fn cli() -> Command {
         )
         .arg_targets_all(
             "Test only this package's library unit tests",
-            "Test only the specified binary",
-            "Test all binaries",
+            "Test only the specified binary's unit tests",
+            "Test all binaries's unit tests",
             "Test only the specified example",
             "Test all examples",
-            "Test only the specified test target",
-            "Test all test targets",
+            "Test only the specified target's integration tests",
+            "Test all test targets's integration tests and library unit tests",
```

Comment:
This seems odd so I would want us to confirm it and whether it is expected behavior updating documenation